src/main/resources/META-INF/plugin.xml

```diff
@@ -34,6 +34,9 @@
 
     <actions>
 
+        <action id="GitMachete.SlideInUnmanagedBelowAction"
```

Comment:
All the actions that are not in a `<group>` are listed at the end, pls move for consistency